src/config/.htaccess

```diff
@@ -1,5 +1,27 @@
-deny from all
+# ------------------------------------------------------------------
+# Purpose of file: block any web access to unallowed files
+#                  stored under the /config directory
+# ------------------------------------------------------------------
+
+# Apache 2.2
+<IfModule !mod_authz_core.c>
+    Deny from all
```

Comment:
I basically didn't change the 2.2 rules at all, but only wrapped them into the `IfModule` check.